components/dropdown/src/auro-dropdown.js

```diff
@@ -772,7 +770,11 @@ export class AuroDropdown extends LitElement {
           part="trigger"
           tabindex="${this.tabIndex}"
           ?showBorder="${this.showTriggerBorders}"
-          >
+          role="${ifDefined(this.triggerContentFocusable ? undefined : this.a11yRole)}"
+          aria-expanded="${ifDefined(this.triggerContentFocusable ? undefined : this.isPopoverVisible)}"
+          aria-controls="${ifDefined(this.triggerContentFocusable ? undefined : this.dropdownId)}"
+          aria-labelledby="${ifDefined(this.triggerContentFocusable ? undefined : 'triggerLabel')}"
+        >
```

Comment:
isDefined is not needed for `this.triggerCOntentFocusable`
```suggestion
          role="${this.triggerContentFocusable ? undefined : this.a11yRole}"
          aria-expanded="${this.triggerContentFocusable ? undefined : this.isPopoverVisible}"
          aria-controls="${this.triggerContentFocusable ? undefined : this.dropdownId}"
          aria-labelledby="${this.triggerContentFocusable ? undefined : 'triggerLabel'}"
        >
```

Reply:
In this case, because we are using a ternary, the result is that the attribute will be removed if the value of `triggerContentFocusable` is truthy because the ternary will return `undefined` to the `ifDefined` function call. Meaning if the trigger content is focusable, this attribute will not be applied to the element. The logic here is correct, and functions as expected.